content/en/docs/concepts/services-networking/gateway.md

```diff
@@ -0,0 +1,186 @@
+to the Gateway by the controller. This address is used as a network endpoint for processing traffic
+of backend network endpoints.
+
+See the [Gateway][gateway] reference for a full definition of this API kind.
+
+### HTTPRoute {#api-kind-httproute}
+
+{{< gateway-feature-state for_gateway_version="v1.0" state="stable" >}}
+
+The HTTPRoute kind specifies routing behavior of HTTP requests from a Gateway listener to backend network
+endpoints. For a Service backend, an implementation may represent the backend network endpoint as a Service
+IP or the backing Endpoints of the Service. An HTTPRoute represents configuration that is applied to the
+underlying Gateway implementation. For example, defining a new HTTPRoute may result in configuring additional
+traffic routes into a cloud load balancer or in-cluster proxy server.
+
+A minimal HTTPRoute example:
+
+{{% code_sample file="service/networking/httproute.yaml" %}}
+
+In this example, HTTP traffic from Gateway `example-gateway` with the Host: header set to `www.example.com`
+and the request path specified as `/login` will be routed to Service `example-svc` on port `8080`.
+
+See the [HTTPRoute][httproute] reference for a full definition of this API kind.
+
+## Request flow
+
+Here is a simple example of HTTP traffic being routed to a Service by using a Gateway and an HTTPRoute:
+
+{{< figure src="/docs/images/gateway-request-flow.svg" alt="gateway-request-flow" class="diagram-medium" link="https://raw.githubusercontent.com/kubernetes/website/main/content/en/docs/images/gateway-request-flow.svg" >}}
+
+_GatewayClass is ommitted in the figure for brevity purposes._
+
+In this example, the request flow for a Gateway implemented as a reverse proxy is:
+
+1. The client starts to prepare an HTTP request for the URL `http://www.example.com`
+2. The client's DNS resolver queries for the destination name and learns a mapping to
+   one or more IP addresses associated with the Gateway.
+3. The client sends a request to the Gateway IP address; the reverse proxy receives the HTTP
+   request and uses the Host: header to match a configuration that was derived from the Gateway
+   and associated HTTPRoute.
+4. Optionally, the reverse proxy can perform request header and/or path matching based
+   on match rules of the HTTPRoute.
+5. Optionally, the reverse proxy can modify the request; for example, to add or emove headers,
+   based on filter rules of the HTTPRoute.
+6. Lastly, the reverse proxy forwards the request to one or more backends.
+
+## Conformance
+
+Gateway API covers a broad set of features and is widely implemented. This combination requires
+clear conformance definitions and tests to ensure that the API provides a consistent experience
+wherever it is used.
+
+See the [conformance][conformance] documentation to understand details such as release channels,
+conformance testing, and feature support levels.
```

Comment:
I'd omit this. If it's important, link from the What's next section to a page about conformance testing for Gateway API.